src/goto-programs/remove_virtual_functions.cpp

```diff
@@ -227,6 +211,35 @@ void remove_virtual_functionst::remove_virtual_function(
   target->make_skip();
 }
 
+void remove_virtual_functionst::get_child_functions_rec(
+  const irep_idt &this_id,
+  const symbol_exprt &last_method_defn,
+  const irep_idt &component_name,
+  functionst &functions)
+{
+  auto findit=class_hierarchy.class_map.find(this_id);
+  if(findit==class_hierarchy.class_map.end())
+    return;
+
+  for(const auto & child : findit->second.children)
+  {
+    exprt method=get_method(child, component_name);
+    functiont function;
```

Comment:
to avoid copying, push_back(functiont()) first and then get a reference to .back()